src/components/SecondaryPanes/Frames/Group.js

```diff
@@ -105,48 +107,58 @@ export default class Group extends Component<Props, State> {
 
     return (
       <div className="frames-list">
-        {group.map(frame => (
-          <FrameComponent
-            copyStackTrace={copyStackTrace}
-            frame={frame}
-            frameworkGroupingOn={frameworkGroupingOn}
-            hideLocation={true}
-            key={frame.id}
-            selectedFrame={selectedFrame}
-            selectFrame={selectFrame}
-            shouldMapDisplayName={false}
-            toggleBlackBox={toggleBlackBox}
-            toggleFrameworkGrouping={toggleFrameworkGrouping}
-            displayFullUrl={displayFullUrl}
-            getFrameTitle={getFrameTitle}
-            disableContextMenu={disableContextMenu}
-          />
-        ))}
+        {group.reduce((acc, frame) => {
+          if (selectable) {
+            acc.push(<FrameIndent />);
```

Comment:
This allow us to have a double indentation for React frames, so the text version is more readable: 

```
Warning: Each child in an array or iterator should have a unique "key" prop. react-dom.development.js:520
    React 10
        warningWithoutStack
        warning
        warnForMissingKey
        warnOnInvalidKey
        reconcileChildrenArray
        reconcileChildFibers
        reconcileChildren
        finishClassComponent
        updateClassComponent
        beginWork
    <anonymous> index.js:6
    js bundle.js:109
    __webpack_require__ bundle.js:20
    <anonymous> bundle.js:84
    <anonymous> bundle.js:1

```